src/test/java/pages/LoginPage.java

```diff
@@ -41,8 +42,8 @@ public LoginPage enterPassword(String password) throws InterruptedException {
     }
 
     public HomePage login() throws InterruptedException {
```

Comment:
**Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.**
Using Thread. sleep() frequently in an automation framework is not a good practice. 

_Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code._

![image](https://user-images.githubusercontent.com/104761482/205234082-8c50e51a-6153-4b85-87d8-ac83c1188d77.png)